pfcpiface/main.go

```diff
@@ -6,11 +6,13 @@ package main
 import (
 	"encoding/json"
 	"flag"
+	"fmt"
```

Comment:
Consistently use log?

Reply:
fmt is now only used to create error log. other places using log.